src/Symfony/Component/HttpKernel/Profiler/FileProfilerStorage.php

```diff
@@ -309,7 +315,14 @@ private function removeExpiredProfiles(): void
         }
 
         while ($line = fgets($handle)) {
-            [$csvToken, , , , $csvTime] = str_getcsv($line);
+            $values = str_getcsv($line);
+
+            if (count($values) < 4) {
```

Comment:
7 is still the correct number of items in a well formatted:
```suggestion
            if (\count($values) < 7) {
```